test.js

```diff
@@ -88,6 +88,41 @@ describe('Test cid.js', function() {
     });
   });
 
+    it('Should respect API responding alternateUrl', function(done) {
```

Comment:
Also a test without token?

Reply:
Sure, always good to have more test coverage : )

Reply:
actually I think that's a duplicate test. test on AMP_TOKEN is already there, we only want to add test to `alternateUrl`. But I added the test anyway : )